src/marqo/README.md

```diff
@@ -36,6 +36,35 @@ __Notes__:
 - This is for marqo-os (Marqo OpenSearch) running locally. You can alternatively set
 `OPENSEARCH_URL` to  a remote Marqo OpenSearch cluster 
 
+## Redis set up (Needed for Options A and E)
+Marqo uses redis to handle concurrency throttling. Redis is automatically set up when running Marqo in docker (Options B-D), but if you are running Marqo locally on your machine (Options A and E), you have to set redis up yourself.
```

Comment:
We don't actually need Redis to be running for development right? It's more of an optional thing - if they want to play around with the throttling/have a closer experience to the final product